docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+############ Create Org2 Identities ######################
+##########################################################
++ cryptogen generate --config=./organizations/cryptogen/crypto-config-org2.yaml --output=organizations
+org2.example.com
++ res=0
++ set +x
+##########################################################
+############ Create Orderer Org Identities ###############
+##########################################################
++ cryptogen generate --config=./organizations/cryptogen/crypto-config-orderer.yaml --output=organizations
++ res=0
++ set +x
+```
+
+However, `network.sh` also provides the option to bring up the network using
+Certificate Authorities (CAs). In a production network, each organization would
+operate a CA (or multiple intermediate CAs) that creates the identities that
+belong to their organization. All the identities that are created by the organizations
+CA would share the same root of trust. Although it takes more time to run the
+test network using CAs than to use cryptogen, bringing up a network using CAs
+can provide you with an introduction to how you would deploy a production network.
+Standing up the Fabric CAs also provides you with the ability to enroll a client
+identity using the Fabric SDKs and create a certificate and private key that can
+be used by your application. Both cryptogen and the Fabric CAs generate the
+crypto material for each organization in the `organizations` folder.
+
+If you would like to bring up a network using Fabric CAs, first run the following
+command to bring down any running networks:
+```
+./network.sh down
+```
+
+You can then bring up the network with the CA flag:
+```
+./network.sh up -ca
+```
+
+After you issue the command, you can see the script bringing up three CAs, one
+for each organization in the network.
+```
+##########################################################
+##### Generate certificates using Fabric CA's ############
+##########################################################
+Creating network "net_default" with the default driver
+Creating ca_org2    ... done
+Creating ca_org1    ... done
+Creating ca_orderer ... done
+```
+
+The script then uses the Fabric CA client to register the users that belong to
+each organization and generate the certificates and keys for each identity. You
+can find the commands that are used to set up the network in the `registerEnroll.sh`
+script in the `organizations/fabric-ca` directory. To learn more about how you
+would use the Fabric CA to deploy a Fabric network, visit the
+[Fabric CA operations guide](Link to guide.html).
+
+## Bring Down the Network
+
+When you are finished using the test network, you can bring down the network
+with the following command:
+```
+./network.sh down
+```
+
+The command will stop and remove the node containers, remove the crypto material
+and the system channel genesis block, and delete the chaincode images from your
+Docker Registry.
```

Comment:
This seems low to me, particularly since you have the command above as part of the CA flow.

Reply:
done